bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -72,10 +72,11 @@
       {
         "uiName": "Writer schema version",
         "fieldName": "writerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
Do not we have any issue while importing earlier streamline topologies as they may not have this configuration?